src/web/routes/application/flow-control/session-accessors/session-accessors.js

```diff
@@ -41,12 +41,22 @@ const getStateFromSession = getJourneySessionProp(STATE_KEY)
 
 const getJourneysFromSession = compose(toPairs, pathOr({}, JOURNEYS_PATH))
 
+const getAdditionalDataForStep = (req, step) => {
+  return pathOr({}, [ADDITIONAL_DATA_KEY, step.path], req.session)
```

Comment:
Could add `session` to the path:
```
const getAdditionalDataForStep = (req, step) =>
  pathOr({}, [ADDITIONAL_DATA_KEY, step.path, 'session'], req)
```

Reply:
I did start that way, but wasn't quite happy referring to `'session'` when the rest of the code refers to `req.session`